src/atomic64.rs

```diff
@@ -197,6 +204,24 @@ impl Atomic for AtomicU64 {
     }
 }
 
+impl AtomicU64 {
+    /// Get the value with the provided memory ordering.
+    pub fn compare_and_swap(&self, current: u64, new: u64, ordering: Ordering) -> u64 {
+        self.inner.compare_and_swap(current, new, ordering)
+    }
+
+    /// Increment the value by a given amount with the provided memory ordering.
+    pub fn inc_by_with_ordering(&self, delta: u64, ordering: Ordering) {
+        self.inner.fetch_add(delta, ordering);
+    }
+
+    /// Swap the current value with the given value, returning that previously
```

Comment:
s/returning that/returning the/

Reply:
:+1: copied the `std` doc comment now.

https://doc.rust-lang.org/std/sync/atomic/struct.AtomicU64.html#method.swap